services/ui-waflog-s3-bucket/serverless.yml

```diff
@@ -4,12 +4,19 @@ frameworkVersion: "2"
 
 plugins:
   - serverless-stack-termination-protection
+  - serverless-deployment-bucket
+  - serverless-s3-remover
 
 provider:
   name: aws
   runtime: nodejs12.x
   region: us-east-1
 
+  deploymentBucket:
+    name: ${self:service}-${opt:stage, self:provider.stage}-deployment-bucket
```

Comment:
If you're still looking for a unique prefix, ive got something really need you can try...
See:  https://newbedev.com/how-to-get-aws-account-id-as-custom-variable-in-serverless-framework

Essentially you run a small piece of javascript, from a file you put inside the service, and you point your variable to it using the syntax there.

So you could drop your javascript fie in... the set the deployment buckets name to 
```
s3:    
  slsbucket:      
    name: ${file(../myCustomFile.js):getAccountId}-${self:service}-${opt:stage, self:provider.stage}-deployment-bucket
```
^^ If the article is correct, that will be able to pull your account ID when the file is first evaluated.

Reply:
I have removed the S3-Remover plugin and added the javascript file for account ID as requested.
Please review.